pkg/virt-controller/watch/vmi.go

```diff
@@ -2239,8 +2240,8 @@ func (c *VMIController) getFilesystemOverhead(pvc *k8sv1.PersistentVolumeClaim)
 }
 
 func (c *VMIController) canMoveToAttachedPhase(currentPhase virtv1.VolumePhase) bool {
-	return currentPhase == "" || currentPhase == virtv1.VolumeBound || currentPhase == virtv1.VolumePending ||
-		currentPhase == virtv1.HotplugVolumeAttachedToNode
+	return (currentPhase == "" || currentPhase == virtv1.VolumeBound || currentPhase == virtv1.VolumePending ||
+		currentPhase == virtv1.HotplugVolumeAttachedToNode) && currentPhase != virtv1.VolumeReady
```

Comment:
This seems kind of weird, how does adding `currentPhase != virtv1.VolumeReady` change anything?  `currentPhase` cannot be any of the explicit values checked AND `virtv1.VolumeReady`.

I think a switch statement with all the valid values enumerated may be the cleanest